Libraries/Components/TextInput/TextInputTypes.js

```diff
@@ -0,0 +1,532 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type React from 'React';
+import type {SyntheticEvent} from 'CoreEventTypes';
+import type {TextStyleProp, ViewStyleProp} from 'StyleSheet';
+import type {ColorValue} from 'StyleSheetTypes';
+import type {ViewProps} from 'ViewPropTypes';
+import type {TextInputType} from 'TextInputNativeComponent';
+import type DocumentSelectionState from 'DocumentSelectionState';
+
+export type Event = Object;
```

Comment:
I would like to fill out the different event types (and the callbacks that use them) but would like to do that in a separate diff